config/maintainers.json

```diff
@@ -10,6 +10,36 @@
       "link_url": null,
       "name": null,
       "show_on_website": false
+    },
+    {
+      "alumnus": false,
```

Comment:
There's been some back-and-forth about the "alumnus" key, but the default now is to either make it 'null' or remove it entirely for those who are not alumni.

https://github.com/exercism/docs/blob/master/maintaining-a-track/maintainer-configuration.md#alumnus

Reply:
Will remove it